src/app/container/paramfiles/paramfiles.service.ts

```diff
@@ -32,7 +32,7 @@ export class ParamfilesService {
     if (type === 'workflows') {
       return this.workflowsService.getTestParameterFiles(id, versionName);
     } else {
-      return this.containersService.getTestParameterFiles(id, versionName, descriptor);
+      return this.containersService.getTestParameterFiles(id, descriptor, versionName);
```

Comment:
Why is this reversing in this PR?

Reply:
Looks like swagger codegen or the swagger.yaml reversed the parameters.

Reply:
It doesn't seem like either or those are changing in this PR though, so was this previous broken and undetected or was there a non-checked-in change here?

Reply:
Previous broken and undetected

Reply:
Ok, might be time to think of a test or extend the parameter ordering that was put in earlier.
This seems like an easy way to introduce an error without noticing